firebase-config/src/androidTest/java/com/google/firebase/remoteconfig/FirebaseRemoteConfigIntegrationTest.java

```diff
@@ -60,7 +60,7 @@ public class FirebaseRemoteConfigIntegrationTest {
   @Mock private ConfigCacheClient mockDefaultsCache;
   @Mock private ConfigFetchHandler mockFetchHandler;
   @Mock private ConfigGetParameterHandler mockGetHandler;
-  @Mock private ConfigMetadataClient metadataClient;
+  @Mock private ConfigSharedPrefsClient metadataClient;
```

Comment:
i might suggest renaming this in a separate PR, and also if we update the class name then the variable should change too? `ConfigSharedPrefsClient sharedPrefsClient`

Reply:
Sure, I will create a separate PR for this.

Reply:
I have created the pull request for this change: https://github.com/firebase/firebase-android-sdk/pull/6440